infra/feast-operator/api/v1alpha1/featurestore_types.go

```diff
@@ -142,6 +151,13 @@ type OnlineStore struct {
 	// Allowed values: "debug", "info", "warning", "error", "critical".
 	// +kubebuilder:validation:Enum=debug;info;warning;error;critical
 	LogLevel string `json:"logLevel,omitempty"`
+
+	// VolumeMounts defines the list of volumes that should be mounted into the container.
+	// This allows attaching persistent storage, config files, secrets, or other resources
+	// required by the Feast components. Ensure that each volume mount has a corresponding
+	// volume definition in the Volumes field.
```

Comment:
let's make the description something more Feast specific... similar to what was done for `Volumes`.